backend/tests/integration/alembic/conftest.py

```diff
@@ -24,6 +25,8 @@ def database(appctx):
 
 
 def clean_db(db):
+    if not database_exists(str(db.engine.url)):
+        create_database(str(db.engine.url))
```

Comment:
isn't it in the `db` fixture?

Reply:
It is in db_cleanup in cleanups. but we are not using them in alembic, we use this fixture. So if you run only alembic tests on empty db it will fail.